tests/pyccel/test_pyccel.py

```diff
@@ -71,14 +71,100 @@ def compile_pyccel(path_dir,test_file, options = ""):
 
 #------------------------------------------------------------------------------
 def compile_c(path_dir,test_file,dependencies,is_mod=False):
```

Comment:
Please add an empty space after each comma (multiple places throughout this file).

Reply:
Is there anyway you can see your way to excusing this for once? Or fixing it for me? Or finding an intern to fix it?

On vim it would just be:
```
:%s/,\W/, /g
:%s/ \+//g
```
But on a phone it will be very tricky

Reply:
Fixed in [1bc652e](https://github.com/pyccel/pyccel/pull/1477/commits/1bc652ec228730c20cfd6daa40b457e7318a2ee5)